suites/pacific/upgrades/tier-1_upgrade_test-4x-to-5x-container.yaml

```diff
@@ -157,3 +157,8 @@ tests:
         config-file-name: test_multitenant_access.yaml
         timeout: 300
     desc: Perform rgw tests
+
+- test:
+    name: test_cephadm_shell.py
```

Comment:
@MohitBis Provide the proper test name not the module name. add polarion id  as well here.
Provide PR link in polarion automation script field and update polarion automation field.

Reply:
Thanks Sunil.

As per the comment.
1. Changed test file name to  "test_cephadm_shell_run.py".
2. Update PR in polarion same added in description.

Reply:
@sunilangadi2 
As per the discussion.
1. Added new method(getting_non_mon_node) to support non-mon node selection.
2. Updated polarion id.

Reply:
Thanks, @MohitBis Changes LGTM
Can you please add test run logs here with new changes?

Reply:
Thanks @sunilangadi2 
Please find below test run logs for new changes
http://magna002.ceph.redhat.com/cephci-jenkins/cephci-run-GBK7R5